inspirehep/utils/citations.py

```diff
@@ -61,3 +61,23 @@ def get_and_format_citations(record):
         result.append(row)
 
     return result
+
+
+def get_citations_from_es(record, page=1, size=10):
```

Comment:
I just noticed! We already have a ``utils`` module here: https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/records/utils.py which are specific for the records